autoread.js

```diff
@@ -31,16 +31,23 @@ function goToNext(){
 		sectionLines[currentLineIndex].style.backgroundColor="lightyellow";
 		currentLineText = sectionLines[currentLineIndex].innerText;
 	}
+
+	sectionLines[currentLineIndex].scrollIntoView({behavior: "smooth", block: "center", inline: "nearest"});
 }
 
 
 function play(){
 	window.speechSynthesis.cancel();
 	speech.text = currentLineText;
 	window.speechSynthesis.speak(speech);
-	
 }
 
+function playHotKey(event){
+	if (event.key == " " || event.type == "touchstart") {
```

Comment:
On mobile, what happens if you try to scroll? seems to me that the touchstart would trigger and then it would scroll to the next line.

Reply:
we probably only want it to trigger next line on tap. If user is scrolling, let them scroll in peace.

Reply:
Scrolling doesn't get interrupted on mobile from what I can tell in chromes dev tools.
It lets you scroll just fine, and tapping still triggers next line.